src/main/java/chess/domain/validateMove/SourceMoveValidator.java

```diff
@@ -0,0 +1,19 @@
+package chess.domain.validateMove;
+
+public class SourceMoveValidator implements ValidateMove {
+    private ValidateMove next;
+
+    @Override
+    public void setNext(ValidateMove validateMove) {
+        this.next = validateMove;
+    }
+
+    @Override
+    public boolean validate(ValidateData validateData) {
+        if (validateData.getSourcePiece().canMove(validateData.getSource(), validateData.getTarget())) {
```

Comment:
```suggestion
        if (validateData.canMove(validateData.getSource(), validateData.getTarget())) {
```
여기도 이런 방식으로 진행할 수 있지 않나요?